### PR TITLE
Create redirect collection from tumblr post_url's

### PR DESCRIPTION
**Pull request by therewasaguy**
If we move this site to monthlymusichackathon.org, we could move the tumblr, and redirect anyone who visits those old links to the archived tumblr.

- Create a redirect layout that redirects to a tumblr_archive
- Add redirects collection with an entry for each tumblr post
- Add `make_redirects.py` script that makes markdown from `tumblr_archive.json`. We might want to use this for other things down the line

### CODE REVIEW

File: _layouts/redirect.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+layout: default
+---
+
+<meta http-equiv="refresh" content="0; url={{site.tumblr_archive}}{{page.permalink}}" />
+
+<h1>This page has been archived</h1>
+<a href="{{site.tumblr_archive}}{{page.permalink}}" title="redirect">Click here</a> if not redirected.
+
+<script>
+	window.location.href = '{{site.tumblr_archive}}{{page.permalink}}';
```
Comment (trevorknight) [inline _layouts/redirect.html]: The page redirects too quickly to read the message, should we add a delay?

Also, what about saying like, "The list of past events can be found {past events link}, to view the old website, click here {tumblr link}"

**Review by trevorknight — COMMENTED** [review]
Awesome! Thanks so much for realizing this was a problem and finding a solution. 

Just small comments

**Comment by trevorknight** [comment]
I also noticed this plug-in among the github-supported ones:

https://github.com/jekyll/jekyll-redirect-from

Do you think that's a good way to go?

File: _layouts/redirect.html (hunk 1 above)
Comment (therewasaguy) [reply _layouts/redirect.html]: Sure, we could add a delay, I think it's worth adding this additional text.

**Comment by therewasaguy** [comment]
Hey @trevorknight thanks for the tip on this Jekyll plugin. I started looking into it last night and should be quick for me to extend this PR to incorporate it. It's doing pretty much the same thing, but may have advantages down the line, especially if we also want to have multiple url's direct to the same page. More soon...

**Comment by therewasaguy** [comment]
@trevorknight it looks like redirects are already in place for posts with https://github.com/musichackathon/mmh_jekyll/commit/bde9f996fc4865a13ad1f20298d44157f55d34b7#diff-f1f667fb320cbdac491bfab20a722dc2R10, cool! 

I'll close this and just add `redirect_from`'s for events

**Pull request closed without merging**